swap/actions.go

```diff
@@ -356,20 +357,15 @@ func (c *CreateSwapOutFromRequestAction) Execute(services *SwapServices, swap *S
 		return swap.HandleError(errors.New("insufficient walletbalance"))
 	}
 
-	/*
-		 feeSat, err := services.policy.GetMakerFee(swap.Amount, swap.OpeningTxFee)
-		 if err != nil {
-
-		return Event_ActionFailed
-		}
-	*/
+	// Parse memo
```

Comment:
Minor nit: "Parse" means interpret, you're doing the opposite, a formatted print